server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -471,6 +472,25 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                 }
             }
             builder.endArray();
+            if (isClone()) {
```

Comment:
Is this related to this pr?

Reply:
Sort of, it's of necessary to make assertion failure messages useful, that's where I noticed this was missing.

Reply:
That looks strange that `Entry` implements `ToXContent` just for rendering assertion error messages? I guess it's fine but I find it strange (and it should implement `ToXContentObject`)